src/rail_gui.cpp

```diff
@@ -2052,6 +2052,7 @@ DropDownList GetRailTypeDropDownList(bool for_replacement, bool all_option)
 	Dimension d = { 0, 0 };
 	/* Get largest icon size, to ensure text is aligned on each menu item. */
 	if (!for_replacement) {
+		used_railtypes &= _railtypes_hidden_mask;
```

Comment:
Shouldn't it be
```suggestion
		used_railtypes &= ~_railtypes_hidden_mask;
```
?

Reply:
Yes